exercises/concept/party-robot/.docs/introduction.md

```diff
@@ -42,4 +42,4 @@ fmt.Sprintf("%.2f", number)
 // Returns: 4.32
 ```
 
-`fmt` contains other functions for working with strings, such as `Println` which simply prints the arguments it receives to the console and `Printf` which formats the input in the same way as `Sprintf` before printing it.
+`fmt` contains other functions for working with strings, such as `Println` which simply prints the arguments it receives to the console and `Printf` which formats the input in the same way as `Sprintf` before printing it. Note that `Println` has built-in whitespace and newline logic when printing output to the console, whereas `Printf` does not.
```

Comment:
This is difficult to understand for someone who doesn't know what that logic is already. I would recommend concretely list the main differences (maybe with a bullet list even).

Also please put every new sentence on a new line (it looks the same when rendered but results in better diffs e.g. here for the reviews.)